app/infra/Docker-Compose.yml

```diff
@@ -27,9 +27,11 @@ services:
   models:
     container_name: models
     # https://hub.docker.com/r/_/mongo/
-    image: tensorflow/tensorflow:latest
+    image: jupyter/tensorflow-notebook:latest
     networks:
       - tfm-iabd-net
+    ports:
```

Comment:
So, you're exposing the ports to the public? Or will they be used just in the Docker Compose's network